cgo_flags.go

```diff
@@ -5,8 +5,8 @@ package rocksdb
 
 // #cgo CPPFLAGS: -Iinternal -Iinternal/include -Iinternal/db -Iinternal/util
 // #cgo CPPFLAGS: -Iinternal/utilities/merge_operators/string_append
-// #cgo CPPFLAGS: -I../c-snappy/internal
-// #cgo CPPFLAGS: -DROCKSDB_PLATFORM_POSIX -DNDEBUG -DSNAPPY
+// #cgo CPPFLAGS: -I../c-snappy/internal -I../c-lz4/internal/lib
```

Comment:
I don't think that matters. The glockfile still points to the c-rocksdb that doesn't require lz4. Am I missing something?